frontend/src/pages/tracker-report.module.scss

```diff
@@ -0,0 +1,284 @@
+@import "../styles/tokens.scss";
+@import "~@mozilla-protocol/core/protocol/css/includes/lib";
+
+.wrapper {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+@keyframes fadein {
+  from {
+    opacity: 0;
+  }
+  50% {
+    // Let the loading screen stay hidden for 200ms;
+    // if it gets removed in that time (because the report has initialised),
+    // showing a "loading" message is just distracting:
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+.loading {
+  padding: $spacing-xl;
+  height: 100%;
+  background-color: $color-grey-05;
+  animation-name: fadein;
+  animation-duration: 400ms;
+}
```

Comment:
Heh, awesome that you're on the lookout for this now :)

I'm guessing that this is fine, though, since MDN [also talks about](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion#examples) "toning down" rather than disabling an animation. But I've [asked](https://mozilla.slack.com/archives/C4E0W8B8E/p1655883522124139) in #accessibility so that we'll know for sure going forward :)

Reply:
Looks like [it's probably not necessary](https://mozilla.slack.com/archives/C4E0W8B8E/p1655910077372399?thread_ts=1655883522.124139&cid=C4E0W8B8E).